pkg/operators/v1alpha1/subscription_types.go

```diff
@@ -118,8 +118,18 @@ const (
 	// SubscriptionBundleUnpackFailed indicates that the unpack job failed
 	SubscriptionBundleUnpackFailed SubscriptionConditionType = "BundleUnpackFailed"
 
-	// SubscriptionOperatorDeprecated indicates that the Operator currently installed with this Subscription has been deprecated.
+	// SubscriptionOperatorDeprecated is a roll-up condition which indicates that the Operator currently installed with this Subscription
+	//has been deprecated. It will be present when any of the three deprecation types (Package, Channel, Bundle) are present.
 	SubscriptionOperatorDeprecated SubscriptionConditionType = "OperatorDeprecated"
```

Comment:
WDYT about renaming this one to just `Deprecated`?

I'm concerned that `OperatorDeprecated` sounds too much like `BundleDeprecated` and might confuse users.

Reply:
Agreed, I've updated to `Deprecated`